composer.json

```
@@ -30,6 +30,7 @@
         "rector/extension-installer": "^0.11.1",
         "rector/rector-cakephp": "dev-main",
         "rector/rector-doctrine": "dev-main",
+        "rector/rector-generator": "^0.4.7",
```

please change to `dev-main` as well for consistency:

```suggestion
        "rector/rector-generator": "dev-main"
```